include/input.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include "raylib.h"
+#include "module.hpp"
+#include <array>
+#include <vector>
+#include <string>
+#include <unordered_map>
+
+#define MAX_MODIFIERS 2
```

Comment:
Nigdy nie define, that way lies maddness. `constexpr size_t MAX_MODIFIERS = 2;`

Reply:
poprawione